client/rpc-spec-v2/src/chain_head/api.rs

```diff
@@ -100,7 +100,7 @@ pub trait ChainHeadApi<Hash> {
 		&self,
 		follow_subscription: String,
 		hash: Hash,
-		key: String,
+		items: Vec<StorageQuery<String>>,
 		child_key: Option<String>,
```

Comment:
The current rendered spec (here: https://paritytech.github.io/json-rpc-interface-spec/api/chainHead_unstable_storage.html) uses `childTrie` instead of `childKey`; does this name need updating or is this change based on an unmerged PR to the spec?

Reply:
Have changed this to `childTrie` instead, missed that from the spec! Thanks!